docs/SystemInAppFunctions.md

```diff
@@ -0,0 +1,51 @@
+# System In-App Functions
+CleverTap iOS SDK 7.2.0+ supports a set of built-in in-app functions. They can be assigned to in-app notification button actions or can be used as stand-alone campaigns. To learn more about them see the CleverTap documentation.
```

Comment:
can we use something like this with hyperlinks: 
"The CleverTap iOS SDK (v7.2.0 and above) supports a set of built-in system in-app functions. These can be triggered either via in-app notification button actions or launched as stand-alone campaigns. For more details, refer to the [CleverTap Documentation](#)."

Reply:
We don't have docs page for it, which link to add here?

Reply:
Done